frontend/src/routes/Search.tsx

```diff
@@ -205,6 +254,55 @@ const SearchEvent: React.FC<SearchEventProps> = ({
 
     return (
         <Item key={id} link={link}>
+            <WithIcon Icon={LuPlayCircle} hideIconOnMobile>
+                <div css={{
+                    color: COLORS.neutral90,
+                    marginRight: "clamp(12px, 4vw - 13px, 40px)",
+                    display: "flex",
+                    flexDirection: "column",
+                    minWidth: 0,
+                }}>
+                    <h3 css={{
+                        color: COLORS.primary0,
+                        marginBottom: 6,
+                        fontSize: 17,
+                        lineHeight: 1.3,
+                        ...ellipsisOverflowCss(2),
+                    }}>{title}</h3>
+                    <Creators creators={creators} css={{
+                        ul: {
+                            display: "inline-block",
+                            overflow: "hidden",
+                            textOverflow: "ellipsis",
+                            whiteSpace: "nowrap",
+                        },
+                        li: {
+                            display: "inline",
+                        },
+                    }} />
+                    {description && <SmallDescription
+                        text={description}
+                        lines={3}
+                    />}
+                    {seriesTitle && seriesId && <div css={{
+                        fontSize: 14,
+                        marginTop: "auto",
+                        paddingTop: 8,
+                        overflow: "clip",
+                        overflowClipMargin: 3,
```

Comment:
Well of course this doesn't work in everyones favorite browser. (I mean safari - which also has trouble with the focus border of the search items)

Reply:
Huff yeah that looks weird. But also the focus outline on the item itself. Seems like the `<li>` or sth has `overflow: hidden`? That shouldn't be necessary? Alternatively, you can just ignore fancy things like `clip-margin` and just add more padding? Does just using `overflow-x` improve stuff? `overflow-clip-margin` support indeed seems to be not great :/

Reply:
Yeah we can just add some padding. Though you better not use `shift+tab` in safari, unless you want to be sad. It seems that the focus outline weirdness of the item itself is caused by the solution that I use for the nested link. It's so annoying that safari can't handle that. Either I look for another solution, or we remove the link again. In which case it also doesn't need to be tabbable.